recipes/r-ukbrapr/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set version = "0.2.9" %}
+{% set sha256 = "693c0ab85d21bea722d6f389898d75eddc3201e07bfb3b3a9e208290dd914d6e" %}
+
+package:
+  name: r-ukbrapr
+  version: {{ version }}
+
+source:
+  - url: https://github.com/lcpilling/ukbrapR/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
```

Let's make this literal:

```suggestion
    sha256: 693c0ab85d21bea722d6f389898d75eddc3201e07bfb3b3a9e208290dd914d6e
```